src/app/components/Results/ResultsList.jsx

```diff
@@ -105,6 +115,10 @@ class ResultsList extends React.Component {
           <span className="nypl-results-room">{location}</span>
           <span className="nypl-results-use">{usageType}</span>
         </div>
+        {
+          (items.length === 1) ?
```

Comment:
Oh no, sorry. The shortcut is fine and preferred. I was just asking you to make sure that you needed the `!!`. I checked it out and it's okay if you have it or not. So you can do 
`{(items.length === 1) && <ItemTable ... />}`